pkgs/development/python-modules/fsspec/default.nix

```diff
@@ -14,14 +14,14 @@
 
 buildPythonPackage rec {
   pname = "fsspec";
-  version = "2021.06.0";
+  version = "2021.07.0";
```

Comment:
atleast python3Packages.s3fs has to be updated since the current version depends on 06 and master depends on 07